README.md

```diff
@@ -108,6 +108,11 @@ to see the overall progress of all translations, and which translations are miss
 
 ## Similar projects
 
+- [Better TlDr](https://github.com/emilyseville7cfg-better-tldr)
+  allows you to write almost the same pages as TlDr but in a standardized way
```

Comment:
> almost the same pages as TlDr

I wouldn't call it "almost the same", just "similar".

Reply:
Fixed.